engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-info.html

```diff
@@ -6,6 +6,12 @@
   </span>
 </div>
 
+<div bst-alert="warning" ng-if="activationKey.multi_content_view_environment">
+  <span translate>
+    This activation key has multiple content view environments, which are not yet displayed in the web UI. Changing the content view or lifecycle environment here will overwrite the activation key's multiple environments. To avoid this, update the activation key via Hammer.</a>
```

Comment:
Should hammer be capitalized?

Reply:
Yes, I think it's correct to capitalize when referring to Hammer, the CLI tool. But if it were part of a `hammer` command, it wouldn't be capitalized.